### PR TITLE
Sizing

### PR DESCRIPTION
**Pull request by mikefrancis**
Address some sizing issues as well as some panel/button tweaks for the Installs tool.

### CODE REVIEW

File: src/scss/components/_navbar.scss [hunk 1]
```diff
@@ -89,7 +75,7 @@ $navDarkBackgroundColour: #3E3C42;
     background-color: #4F4D54;
     border-bottom: 0;
 
-    .navbar__brand {
+    .brand {
```
Comment (edcs) [inline src/scss/components/_navbar.scss]: `hero.html` is still using the old class name.

Comment (mikefrancis) [inline src/scss/components/_navbar.scss]: GAH

Comment (edcs) [inline src/scss/components/_navbar.scss]: `navbar.html` too :flushed:

Comment (mikefrancis) [inline src/scss/components/_navbar.scss]: Is it? Not on [here](https://github.com/Pod-Point/frontend-framework/blob/fix/sizing/docs/navbar.html#L13) for me

**Comment by edcs** [comment]
:+1:

**Pull request merged**